aries-site/src/examples/templates/global-header/components/HeaderNav.js

```diff
@@ -0,0 +1,84 @@
+import React, { useContext, useState } from 'react';
+import {
+  Avatar,
+  Box,
+  Button,
+  DropButton,
+  Nav,
+  ResponsiveContext,
+  Text,
+} from 'grommet';
+import { HelpOption, Home } from 'grommet-icons';
+import { UserContext } from '.';
+
+export const HeaderNav = () => {
+  const size = useContext(ResponsiveContext);
+  const { user } = useContext(UserContext);
+  const [open, setOpen] = useState();
+
+  return user ? (
+    <Nav align="center" direction="row" gap="small">
+      {size !== 'small' && (
+        <>
+          <Button icon={<HelpOption />} a11yTitle="Help" title="Help" />
+          <Button icon={<Home />} a11yTitle="Home" title="Home" />
+        </>
+      )}
+      <DropButton
+        dropContent={<AvatarDropContent />}
+        dropProps={{ align: { top: 'bottom', right: 'right' } }}
+        open={open}
+        onOpen={() => setOpen(true)}
+        onClose={() => setOpen(false)}
+      >
+        {!user.image ? (
+          <Avatar background="blue!" flex={false}>
+            <Text size="large" color="text-strong">
+              JD
+            </Text>
+          </Avatar>
+        ) : (
+          <Avatar src={user.image} />
+        )}
+      </DropButton>
+    </Nav>
+  ) : null;
+};
+
+const AvatarDropContent = () => {
```

Comment:
Trying to think of something more explicit as to what this content is. Mainly for those who are following this for their own implementations.

UserProfile?
UserInfo?
UserMenu?
ProfileMenu?

Reply:
Going with UserDetails.